python/tk_nuke_writenode/handler.py

```diff
@@ -1111,7 +1111,15 @@ def __populate_format_settings(self, node, file_type, file_settings, reset_all_s
                 # skip this setting:
                 continue
             
-            knob = write_node.knob(setting_name)
+            # handle settings for gizmo's additional nodes
+            setting_tokens = setting_name.split('.')
+            if len(setting_tokens) == 2:
```

Comment:
Not really... It would be necessary if we were to put a gizmo inside that gizmo, but that would be bad practice ;)

And if it's because of a typo in the env, I think we want it to go through the error on line 1124...